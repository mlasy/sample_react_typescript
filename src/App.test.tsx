import App from './App'
import renderer from 'react-test-renderer';
import React from 'react';
import { Provider } from 'react-redux';
import store from './store';

describe('App', () => {
  test('App renders', () => {
    const component = renderer.create(
      <Provider store={store}>
        <App />
      </Provider>
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})