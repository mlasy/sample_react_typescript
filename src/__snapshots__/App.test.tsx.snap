// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App renders 1`] = `
<div
  style={
    Object {
      "margin": "20px",
    }
  }
>
  <input
    onChange={[Function]}
    value=""
  />
  <button
    onClick={[Function]}
  >
    Add
  </button>
  <ul />
</div>
`;
